apps/jenkins/jenkins/sbox-intsvc/jenkins.yaml

```diff
@@ -6,6 +6,9 @@ metadata:
 spec:
   values:
     checkDeprecation: false
+    serviceAccount:
+      create: false
+      name: jenkins
```

Comment:
I don't think you need this it should just work but ok anyway

Reply:
this is the only thing added in the PR - if we remove this then we're using the current config that is what is currently seeing the issue?

Reply:
will give it a whirl

Reply:
it's the `create: false` you need, i don't think you need the name.